### DISCUSSION TITLE
Adding HttpRequest::getCookie(std::string_view key) API?

### DISCUSSION THREAD

**Discussion by ddevienne** [uNetworking/uWebSockets#1677] [Q&A] [answered]
One can of course use `req->getHeader("cookie"sv")` and parse manually, but common enough to warrant a dedicated API?
There may also be percent-encoding, quoting, or other rules which are subtle enough to warrant being done right in a single place?
For now, I'll do the parsing myself of course (ad-hoc, ignoring possible subtleties). Just a thought. Thanks, --DD

PS: `res->setCookie(...)` might also be nice too, and may have a more complex API, not sure

**Comment by uNetworkingAB** [comment, marked as answer]
Yes I agree, it is a feature one would expect
